### PR TITLE
Fixes Plone Site serialization when there is a field with read_permision set

### LINKED ISSUE #1830: Adding a behavior with permissions to a PloneSite breaks the serialization
Traceback
```
2024-10-31 17:43:51,233 ERROR   [Zope.SiteErrorLog:36][waitress-0] AttributeError: http://localhost:8080/Plone/++api++/GET_application_json_
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 181, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 391, in publish_module
  Module ZPublisher.WSGIPublisher, line 285, in publish
  Module ZPublisher.mapply, line 98, in mapply
  Module ZPublisher.WSGIPublisher, line 68, in call_object
  Module plone.rest.service, line 21, in __call__
  Module plone.restapi.services, line 19, in render
  Module plone.restapi.services.content.get, line 16, in reply
  Module plone.restapi.serializer.site, line 88, in __call__
  Module plone.restapi.serializer.site, line 133, in check_permission
AttributeError: 'SerializeSiteRootToJson' object has no attribute 'permission_cache'
```

### PR DESCRIPTION
**Pull request by ericof**
Fixes #1830

----
📚 Documentation preview 📚: https://plonerestapi--1831.org.readthedocs.build/

### CODE REVIEW

**Comment by ericof** [comment]
@jenkins-plone-org please run jobs

File: news/1830.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixes Plone Site serialization when there is a field with read_permission set [@ericof]
```
Comment (davisagli) [inline news/1830.bugfix:1]:
```suggestion
Fixed Plone Site serialization when there is a field with read_permission set. @ericof
```

**Review by davisagli — APPROVED** [review]
Thanks!

**Comment by mauritsvanrees** [comment]
As last change for Plone 6.1.0b1 I have added `plone.restapi` 9.8.3 with this fix.

**Pull request merged**